partner_company_group/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner Company Group",
+    "summary": "Adds the possibility to add a company group to a company",
+    "version": "16.0.1.0.0",
+    "category": "Sales",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
```

Comment:
Could you add myself as maintainer, please?

Just add the following line above this one:

`"maintainers": ["luisg123v"],`